src/cbmc/cbmc_parse_options.cpp

```diff
@@ -662,6 +662,8 @@ int cbmc_parse_optionst::get_goto_program(
     if(!binaries.empty())
       config.set_from_symbol_table(symbol_table);
 
+    config.set_addr_bits(cmdline, symbol_table);
```

Comment:
Well, we can always set defaults for all languages, can't we? If we have a design that requires calling the `config` 1) multiple times and 2) in a certain sequence then this is bound to break.